targets/TARGET_RENESAS/TARGET_RZ_A1XX/TARGET_VK_RZ_A1H/mbed_drv_cfg.h

```diff
@@ -0,0 +1,19 @@
+#ifndef MBED_DRV_CFG_H
```

Comment:
Please add copyrigh header files (as the rest of th files contain it)

Reply:
I will add the copyright in the below header files.
targets/TARGET_RENESAS/TARGET_RZ_A1XX/TARGET_RZ_A1H/mbed_drv_cfg.h 
targets/TARGET_RENESAS/TARGET_RZ_A1XX/TARGET_VK_RZ_A1H/mbed_drv_cfg.h